typing/env.mli

```diff
@@ -170,6 +170,9 @@ val crc_of_unit: string -> Digest.t
 
 val imports: unit -> (string * Digest.t option) list
 
+(* [is_opaque md] returns true if [md] is an opaque imported module  *)
+val is_opaque: string -> bool
```

Comment:
I think `opaque_import` or `is_opaque_import` would be a clearer name. It is not clear from the type *what* is or is not opaque.